net/deluge/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2008-2011 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=deluge
+PKG_VERSION:=1.3.15
+PKG_RELEASE:=1
+PKG_MAINTAINER:= Sergey Pushilin <pushilin@gmail.com>
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=http://download.deluge-torrent.org/source
+PKG_HASH:=a96405140e3cbc569e6e056165e289a5e9ec66e036c327f3912c73d049ccf92c
+
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python-package.mk
+
+define Package/deluge
+	SUBMENU:=BitTorrent
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=A lightweight BitTorrent client
+	URL:=http://deluge-torrent.org
+	DEPENDS:=+python +rblibtorrent +chardet +pyxdg +python-twisted +python-pyopenssl +python-setuptools +python-service-identity
+endef
+
+define Package/deluge/description
+BitTorrent client with a client/server model.
+endef
+
+define Package/deluge/conffiles
+/etc/config/deluge
+endef
+
+define Build/Compile
+	$(call Build/Compile/PyMod,,install --prefix="/usr" --root="$(PKG_INSTALL_DIR)")
+endef
+
+define Package/deluge/install
```

Comment:
if you rename this to `PyPackage/deluge/install`, add a VARIANT:=python to this package, and call `$(eval $(call PyPackage,deluge))`, you can remove some parts from this install rule